docs/docs/device/useAnimatedKeyboard.mdx

```diff
@@ -2,72 +2,112 @@
+#### `options` <Optional />
+
+Optional object containing additional configuration:
 
-Optional object containing additional configuration.
+- `isStatusBarTranslucentAndroid` - removes top inset on Android i.e. to use translucent status bar on Android, set this option to `true`. Defaults to `false`. Ignored on iOS.
 
 ### Returns
 
-Hook `useAnimatedKeyboard` returns an instance of [[AnimatedKeyboardInfo](#animatedkeyboard-object)];
+Hook `useAnimatedKeyboard` returns an object containing these fields:
 
-### Types
+| Name   | Type                         | Description                                                                                                                                                |
+| ------ | ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| height | `SharedValue<number>`        | A [shared value](/docs/fundamentals/glossary#shared-value) containing current height of the keyboard.                                                      |
+| state  | `SharedValue<KeyboardState>` | A [shared value](/docs/fundamentals/glossary#shared-value) containing current state of the keyboard. Possible states: `{ CLOSED, OPEN, CLOSING, OPENING }` |
 
-#### `AnimatedKeyboardInfo: [object]`
+## Example
 
-Properties:
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import AnimatedKeyboardSrc from '!!raw-loader!@site/src/examples/AnimatedKeyboard';
 
-- `height`: [[SharedValue](/docs/core/useSharedValue)] contains `[number]`
-  contains current height of the keyboard
-- `state`: [[SharedValue](/docs/core/useSharedValue)] contains `[enum]`
-  contains current state of the keyboard. Possible states: `{ CLOSED, OPEN, CLOSING, OPENING }`
+<InteractiveExample
+  src={AnimatedKeyboardSrc}
+  component={
+    <ThemedVideo
+      center
+      width={300}
+      sources={{
+        light: '/recordings/useAnimatedKeyboard_light.mov',
+        dark: '/recordings/useAnimatedKeyboard_dark.mov',
+      }}
+    />
+  }
+/>
 
-#### `AnimatedKeyboardOptions: [object]`
+## Remarks
 
-Properties:
+- On Android, make sure to set `android:windowSoftInputMode` in your `AndroidManifest.xml` to `adjustResize`. Then, using the `useAnimatedKeyboard` hook disables
+  the default Android behavior (resizing the view to accommodate keyboard) in the whole app. Using values from `useAnimatedKeyboard` hook you can handle the keyboard yourself. Unmounting all components that use `useAnimatedKeyboard` hook brings back the default Android behavior.
+- On Android, using the `useAnimatedKeyboard` hook expands root view to full screen (immersive mode) and takes control over insets management. It applies the top and bottom margins according to the insets when `isStatusBarTranslucentAndroid` is false and sets top margin to 0 and bottom padding according to the navigation insets when `isStatusBarTranslucentAndroid` is true.
```

Comment:
```suggestion
- On Android, using the `useAnimatedKeyboard` hook expands root view to full screen ([immersive mode](https://developer.android.com/develop/ui/views/layout/immersive)) and takes control over insets management. It applies the top and bottom margins according to the insets when `isStatusBarTranslucentAndroid` is `false` and sets top margin to `0` and bottom padding according to the navigation insets when `isStatusBarTranslucentAndroid` is `true`.
```

Lets add a link to the official android documentation explaining what immersive mode is.

This point could be split into subpoints for easier text skimming eg.
- On Android...
  - when `isStatusBarTranslucentAndroid` is set `true` this happens
  - when `isStatusBarTranslucentAndroid` is set to false that happens